descriptor.md

```diff
@@ -108,7 +108,7 @@ Implementations MUST implement SHA-256 digest verification for use in descriptor
 
 ## Examples
 
-The following example describes a [_Manifest_](manifest.md#image-manifest) with a content identifier of "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270", of size 7682 bytes:
+The following example describes a [_Manifest_](manifest.md#image-manifest) with a content identifier of "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270",  and a size of 7682 bytes:
```

Comment:
You have “,  ” (a comma and two space) where [I think we want][1] a single space.

[1]: https://github.com/opencontainers/image-spec/pull/426#discussion_r85781936